docs/core/introduction.md

```diff
@@ -1,57 +1,379 @@
+### CI/CD
 
-## Support
+MSBuild and the .NET CLI can be used with various continuous integration tools and environments, such as:
+
+* [GitHub Actions](https://github.com/features/actions)
+* [Azure DevOps](/azure/devops/user-guide/what-is-azure-devops)
+* [CAKE](https://cakebuild.net/)
+* [FAKE](https://fake.build/)
+
+For more information, see [Using .NET SDK and tools in Continuous Integration (CI)](tools/using-ci-with-cli.md)
+
+### NuGet
+
+[NuGet](/nuget/what-is-nuget) is an open-source package manager designed for .NET. A NuGet package is a *.zip* file with the `.nupkg` extension that contains compiled code (DLLs), other files related to that code, and a descriptive manifest that includes information like the package's version number. Developers with code to share create packages and publish them to [nuget.org](https://nuget.org) or a private host. Developers who want to use shared code add a package to their project and can then call the API exposed by the package in their project code.
+
+For more information, see [NuGet documentation](/nuget/).
+
+### .NET Interactive
+
+.NET Interactive is a group of CLI tools and APIs that enable users to create interactive experiences across the web, markdown, and notebooks.
+
+For more information, see the following resources:
+
+* [.NET In-Browser Tutorial](https://dotnet.microsoft.com/learn/dotnet/in-browser-tutorial/1)
+* [Using .NET notebooks with Jupyter on your machine](https://github.com/dotnet/interactive/blob/main/docs/NotebooksLocalExperience.md)
+* [.NET Interactive documentation](https://github.com/dotnet/interactive/blob/main/docs/README.md)
+
+## Execution models
+
+.NET apps run [managed code](../standard/managed-code.md) in a runtime environment known as the Common Language Runtime (CLR).
+
+### CLR
+
+The .NET [CLR](../standard/clr.md) is a cross-platform runtime that includes support for Windows, macOS, and Linux. The CLR handles memory allocation and management. The CLR is also a virtual machine that not only executes apps but also generates and compiles code using a just-in-time (JIT) compiler.
+
+For more information, see [Common Language Runtime (CLR) overview](../standard/clr.md).
+
+### JIT compiler and IL
+
+Higher-level .NET languages, such as C#, compile down to a hardware-agnostic instruction set, which is called Intermediate Language (IL). When an app runs, the JIT compiler translates IL to machine code that the processor understands. JIT compilation happens on the same machine that the code is going to run on.
+
+Since JIT compilation occurs during execution of the application, compile time is part of the run time. Therefore, JIT compilers have to balance time spent optimizing code against the savings that the resulting code can produce. But a JIT compiler knows the actual hardware and can free developers from having to ship different implementations for different platforms.
```

Comment:
```suggestion
Since JIT compilation occurs during execution of the application, the compile time is part of the run time. Therefore, JIT compilers have to balance time spent optimizing code against the savings that the resulting code can produce. But a JIT compiler knows the actual hardware and can free developers from having to ship different implementations for different platforms.
```